app/services/articles/feeds/find_featured_story.rb

```diff
@@ -1,9 +1,18 @@
 module Articles
   module Feeds
     module FindFeaturedStory
-      def self.call(stories)
+      # @param stories [ActiveRecord::Relation, #detect] pick a
+      #        featured story from this enumerable
+      # @param must_have_main_image [Boolean] if true, the featured
+      #        story must have a main image
+      # @return [Article]
+      #
+      # @note the must_have_main_image parameter name matches PR #15240
```

Comment:
@jeremyf we should definitely document the params more often, this is very nicely done. ✨

I've never seen @note before in the description of params, is there some link you can point me to to read up the options available?

Reply:
>I've never seen @note before in the description of params, is there some link you can point me to to read up the options available?

https://rubydoc.info/gems/yard/file/docs/Tags.md#note

Reply:
> I'm also not 100% sure what the note means? I did take a peak at PR #15240. Is there any way we can be more specific without redirecting the developer to another PR?

I'm hesitant to include a reference to a named class that is yet to be merged.  Hence I went with a link to the PR.  I have it in my follow-up list to adjust the note and normalize once this and the other PR are merged.

Reply:
Sounds good 👍